templates/terraform/property_documentation.erb

```diff
@@ -14,6 +14,18 @@
 <%   end -%>
 <% end -%>
 <%= indent(property.description.strip.gsub("\n\n", "\n"), 2) -%>
+<% if property.is_a?(Api::Type::Enum) && property.validation.nil? && !property.output -%>
```

Comment:
Can we also get this information into the description that appears in the resource schema? I'd imagine that tools that use that information would probably like it to be there too.

Reply:
Added to the schema in a different format with `"` and `["POSSIBLE_VALUE"]`